redis-cache.php

```diff
@@ -28,6 +28,8 @@
 
 define( 'WP_REDIS_VERSION', $meta['Version'] );
 
+defined( 'WP_REDIS_METRICS_MAX_TIME' ) || define( 'WP_REDIS_METRICS_MAX_TIME', HOUR_IN_SECONDS );
```

Comment:
We're not defining default constants for others like `WP_REDIS_MAXTTL`. Can we just do a check in `class-plugin.php`?

Reply:
introduced a static getter method instead